packages/interaction/src/IHitArea.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Interface for classes that represent a hit area.
```

Comment:
New line giving a couple of examples of what kind of classes. Eg PIXI.Rectangle

Reply:
I update the JSDoc comment as per the review comment.